ztp/kube-compare-reference/optional/ptp-config/PtpConfigBoundary.yaml

```diff
@@ -9,7 +9,7 @@ spec:
   profile:
   - name: "boundary"
     ptp4lOpts: "-2"
-    phc2sysOpts: "-a -r -n 24"
+    phc2sysOpts: "-a -r -n (?<domainNumber>[0-9]+)"
```

Comment:
I have few more suggestions, we can have single ptpconfig for events and non events if we compare it only opts changes . The reason we have different files  because customer never update ptpconfig after they enable events . I can make changes to code to dynamically update options when event is enabled or do you have better method to validate?. 

*** Event **  
    ptp4lOpts: "-2 --summary_interval -4"
    phc2sysOpts: "-a -r -m -n (?<domainNumber>[0-9]+) -N 8 -R 16"
***  Non event  **
     ptp4lOpts: "-2"
    phc2sysOpts: "-a -r -n 24"

Reply:
Also since you are here updating config Can i ask to make these two changes 
Update  MaxInSpecOffset to 1500 for T-GM and we are missing  ptpClockThreshold which is configurable for all types of ptpconfig 

 ptpClockThreshold:
      holdOverTimeout: 1500  
      maxOffsetThreshold: 100
      minOffsetThreshold: -100

Reply:
I really do love the idea of simplifying our config, and if we can automatically add the proper event options when events are enabled, that's probably a great enhancement.

In fact, I think these are all good ideas, but maybe worth their own separate bugs and separate PRs...